sources/updater/waves/default-waves.toml

```
@@ -0,0 +1,18 @@
+# The following represents a "normal" set of update waves for a typical
+# deployment. The deployment lasts for 6 days, and gradually increases the
+# nodes updated at once.
+[[wave]]
+start_after= '2 hours'
```

```suggestion
start_after= '1 hour'
```
(matching above suggestion)